### PR TITLE
Don't generate JavaDoc noise, #28732

### PR DESCRIPTION
**Pull request by patriknw**
Why did we have that enabled?
What will break?

References #28732

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
LGTM, unless it is used somehow i don't think it adds any value

**Comment by patriknw** [comment]
before merging, let me take a look at the output

**Comment by patriknw** [comment]
This is how it looks like with `fabricateParams=true`:
Screenshot 2020-04-29 at 08 30 52

(for some reason it doesn't add it on all methods)

and this is with `false` (this PR):
Screenshot 2020-04-29 at 08 31 11

**Review by raboof — APPROVED** [review]
It seems we added it in https://github.com/akka/akka/commit/18688fc84b518e2968b8710b7ab31c889f9264d6, presumably to make doclint happy, but we disabled doclint again later.

I don't see a reason to enable it though, indeed the differences seem like noise to me as well.

**Pull request merged**
